website/get-involved/index.html

```diff
@@ -34,6 +35,25 @@ <h2>{{ self.page_desc() }}</h2>
 </nav>
 
 <main role="main">
+    <section id="opensource">
+      <h2>{{ _('Why Contribute?') }}</h2>
+      <div class="row">
+            <div class="col full">
+                <p>{{ _('Thunderbird is created by an online community of volunteers.') }}</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <h3>{{ _('What is Open Source Software?') }}</h3>
+                    <p>{{ _('Thunderbird is open source software, meaning that anyone can inspect, modify, enhance it, and share it. But the fun is not just for programmers!
+                        Thunderbird is created by a community of designers, translators, people who do documentation and support, as well as developers. We invite anyone to come
```

Comment:
"people who do documentation" seems awkward. In theory they're called technical writers, but that's a little bit off putting to someone who isn't a professional, so maybe:

"Thunderbird is created by a community of designers, translators, documentation writers and support people in addition to developers."

Reply:
Good feedback, agreed, resolved.